packages/dev/src/main.ts

```diff
@@ -259,6 +259,8 @@ export class NetlifyDev {
   }
 
   async start() {
+    this.#logger.log('Netlify dev environment booting...')
```

Comment:
Don't overanalyze this. We're reviewing logging very soon anyway. This is helpful for now.

Reply:
Can you elaborate? Useful how? Just trying to make sure we don't release a version of the package that has this.